### PR TITLE
[Merged by Bors] - chore: mark neg_div_neg_eq as simp

### PR DESCRIPTION
**Pull request by alexjbest**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/NumberTheory/ModularForms/JacobiTheta/Basic.lean [hunk 1]
```diff
@@ -100,9 +100,11 @@ theorem jacobiTheta_S_smul (τ : ℍ) :
   rw [mul_comm ((1 : ℂ) / _) _, mul_one_div, eq_div_iff ha', mul_comm _ (_ ^ _), eq_comm] at this
   have expo1 : ∀ n : ℤ, -↑π / (-I * ↑τ) * (n : ℂ) ^ 2 = ↑π * I * (n : ℂ) ^ 2 * (-↑τ)⁻¹ := by
     intro n
-    field_simp [hτ, I_ne_zero]
+    simp only [neg_mul, neg_div_neg_eq, div_mul_eq_mul_div, inv_eq_one_div, mul_div_assoc', mul_one,
```
Comment (jcommelin) [inline Mathlib/NumberTheory/ModularForms/JacobiTheta/Basic.lean:103]: Should this be viewed as a regression? Or is there an explanation why this `simp only` is morally the better proof?

**Review by jcommelin — COMMENTED** [review]
bors d+

File: Mathlib/NumberTheory/ModularForms/JacobiTheta/Basic.lean (hunk 1 above)
Comment (alexjbest) [reply Mathlib/NumberTheory/ModularForms/JacobiTheta/Basic.lean:103]: Before it was nonterminal so liable to break

Comment (alexjbest) [reply Mathlib/NumberTheory/ModularForms/JacobiTheta/Basic.lean:103]: But I'm not sure we have a field simp only

**Pull request closed without merging**